source/module_cell/print_cell.cpp

```diff
@@ -23,17 +23,15 @@ namespace unitcell
         bool direct = (Coordinate == "Direct");
         std::string table;
         table += direct? "DIRECT COORDINATES\n": FmtCore::format("CARTESIAN COORDINATES ( UNIT = %20.12f Bohr ).\n", lat0);
-        const std::string redundant_header = direct? "taud_": "tauc_";
-        table += FmtCore::format("%8s%20s%20s%20s%8s%20s%20s%20s\n", "atom", "x", "y", "z", "mag", "vx", "vy", "vz");
+        table += FmtCore::format("%8s%10s%10s%10s%8s%8s%8s%8s\n", "atom", "x", "y", "z", "mag", "vx", "vy", "vz");
         for(int it = 0; it < ntype; it++)
         {
             for (int ia = 0; ia < atoms[it].na; ia++)
             {
                 const double& x = direct? atoms[it].taud[ia].x: atoms[it].tau[ia].x;
                 const double& y = direct? atoms[it].taud[ia].y: atoms[it].tau[ia].y;
                 const double& z = direct? atoms[it].taud[ia].z: atoms[it].tau[ia].z;
-                table += FmtCore::format("%5s%-s%-5d%20.10f%20.10f%20.10f%8.4f%20.10f%20.10f%20.10f\n", // I dont know why there must be a redundant "tau[c|d]_" in the output. So ugly, it should be removed!
-                                        redundant_header, 
+                table += FmtCore::format(" %-s%-5d%10.3f%10.3f%10.3f%8.3f%8.3f%8.3f%8.3f\n", 
```

Comment:
unitcell_test.cpp should also be modified.

Reply:
this has been fixed